Week1/solution1.py.py

```diff
@@ -0,0 +1,10 @@
+n=int(input("enter any number"))
+if n>1:
+	for i in range(2,int(n/2)+1):

```

Comment:
Can you explain me a simple formula of a prime number in a for loop?

Reply:
For i in range(i,number+1):
     if(number%i==0):
          Count+=0
     i+=1
If(count==2):
    Print('prime number')